app/javascript/packs/selectors/answersSelector.js

```diff
@@ -10,7 +10,7 @@ export function getAnswerForQuestionAndBuilding(questionId, buildingId, state) {
 export function getRemainingAnswersforCategory(questions, buildingId, state) {
   return questions.reduce((count, question) => {
     let answer = state.buildings[buildingId].answers[question.id];
-    if (!answer || !answer.text.trim() && !answer.attachment_file_name) {
+    if (!answer || !answer.text && !answer.attachment_file_name) {
```

Comment:
Why was this `.trim()` removed?

Reply:
I'm guessing the reason this was written originally was to check if there was no attachment (text is empty and attachment_file_name is undefined): 
`!answer.text.trim() && !answer.attachment_file_name`
However, for file options, text is actually undefined since we only set the attachment info in our request, so I changed it to check for undefined rather than empty text

Reply:
Is this a similar issue to #202? Answer objects should always have some kind of `text` field, even if it's an empty string (`"".trim()` would still be `""`, which is falsy). In console, I see this as well:

![image](https://user-images.githubusercontent.com/2977329/38546838-88b540e4-3c62-11e8-8c40-97ede81e6ca1.png)

Can you check `window.INITIAL_STATE` to see if `text` is undefined there as well? Otherwise it may be a similar #202 reducer-related issue.